### PR TITLE
Update Go dictionary to Go 1.11.

### PR DESCRIPTION
**Pull request by AlekSi**
Please note that only `go.txt` is updated. `npm run build` does not work for me.

### CODE REVIEW

File: packages/golang/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Change Log
 
+## 1.1.TODO
```
Comment (AlekSi) [inline packages/golang/CHANGELOG.md]: Also this ^

**Comment by Jason3S** [comment]
Thank you

**Comment by AlekSi** [comment]
Please wait a minute, I will send another PR.

**Comment by AlekSi** [comment]
See #33.

**Pull request merged**